database/seed.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-plusplus */
+const mongoose = require('mongoose');
+const Promise = require('bluebird');
+mongoose.Promise = require('bluebird');
+const { Price } = require('./models/prices');
+const { Seller } = require('./models/sellers');
+
+mongoose.connect('mongodb://localhost/seller-catalog', { useMongoClient: true });
+
+// Preparing product documents
```

Comment:
Do we want these comments in the master? If so, maybe keeping it consistent and changing the word `Preparing` to the same word in line 32 `Prepare` vice versa.